database/models/tables.py

```diff
@@ -51,9 +51,26 @@ def _full_search(self, search_keyword):
         abort(404)
 
     def add_new_artist(self, request):
-        
+        #TODO: GenreTable and related ones need to be updated when a new artist is added
```

Comment:
Note to self: TJ's To Do note is really good here. Let's make a GitHub issue out of it.